cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinQuery.java

```diff
@@ -96,11 +105,50 @@ public URI toNvaFacetURI() {
      * @return an URI to NVA (default) Projects with parameters.
      */
     public URI toURI() {
-        return
-            UriWrapper.fromUri(CRISTIN_API_URL)
-                .addChild(getCristinPath())
-                .addQueryParameters(toParameters())
-                .getUri();
+        var uri = UriWrapper.fromUri(CRISTIN_API_URL)
+                      .addChild(getCristinPath());
+
+        uri = appendParameters(toParameters(), uri);
```

Comment:
due to bug in UriWrapper?

Should we rather fix UriWrapper?

Reply:
Fixed in newest version that was merged today, as said in earlier comment. Will fix either in this or next PR